demo.py

```diff
@@ -62,21 +61,22 @@
 #| * figure size:
 
 fig = plt.figure(figsize=(5, 5))
-fig, axes = make_2d_axes(['omegabh2', 'omegach2', 'H0'], fig=fig, tex=mcmc.tex)
+fig, axes = make_2d_axes(['omegabh2', 'omegach2', 'H0'], fig=fig)
 mcmc.plot_2d(axes);
 
 #| * legends:
 
-fig, axes = make_2d_axes(['omegabh2', 'omegach2', 'H0'], tex=mcmc.tex)
+fig, axes = make_2d_axes(['omegabh2', 'omegach2', 'H0'])
 mcmc.plot_2d(axes, label='Posterior');
-axes.iloc[-1, 0].legend(bbox_to_anchor=(len(axes), len(axes)), loc='upper left');
+axes.iloc[0, 0].legend(loc='upper left');
+fig.tight_layout()
 
 #| * unfilled contours  &  modifying individual axes:
 
-fig, axes = make_2d_axes(['omegabh2', 'omegach2', 'H0'], tex=mcmc.tex)
-mcmc.plot_2d(axes.iloc[0:1, :], types=dict(upper='kde', lower='kde', diagonal='kde'), fc=None);
-mcmc.plot_2d(axes.iloc[1:2, :], types=dict(upper='kde', lower='kde', diagonal='kde'), fc=None, cmap=plt.cm.Oranges, lw=3);
-mcmc.plot_2d(axes.iloc[2:3, :], types=dict(upper='kde', lower='kde', diagonal='kde'), fc='C2', ec='C3', c='C4', lw=2);
+fig, axes = make_2d_axes(['omegabh2', 'omegach2', 'H0'])
+mcmc.plot_2d(axes.iloc[0:1, :], kind=dict(upper='kde_2d', lower='kde_2d', diagonal='kde_1d'), fc=None);
+mcmc.plot_2d(axes.iloc[1:2, :], kind=dict(upper='kde_2d', lower='kde_2d', diagonal='kde_1d'), fc=None, cmap=plt.cm.Oranges, lw=3);
+mcmc.plot_2d(axes.iloc[2:3, :], kind=dict(upper='kde_2d', lower='kde_2d', diagonal='kde_1d'), fc='C2', ec='C3', c='C4', lw=2);
```

Comment:
We'll want to raise a warning for the use of `'kde'` instead of `'kde_2d'`, since `'kde'` is the pandas version, whereas we would want to use anesthetic's own `'kde_1d'` and `'kde_2d'`.

Reply:
This is done in 3faf577. I'm going to also put in hints for:
- tex
- ns_output
- fig, axes (if I can)
We should have a think if there are any other gotcha's, and add an issue to eventually remove these hints at another minor release.

Reply:
> We should have a think if there are any other gotcha's, and add an issue to eventually remove these hints at another minor release.

Another few that come to mind:
* kwarg `kind` instead of the previous `types` (2d) or `plot_type` (1d)
* `D_KL` instead of `D`, as well as `d_G` instead of `d` (and hint that there is now also `logL_P`?)

Do we want to add those in this PR?

Reply:
Done in 1ef6cc7